cloud-controller-manager/do/loadbalancers.go

```diff
@@ -936,7 +962,11 @@ func buildTLSForwardingRule(forwardingRule *godo.ForwardingRule, service *v1.Ser
 		// to match the EntryProtocol.
 	} else {
 		forwardingRule.CertificateID = certificateID
-		forwardingRule.TargetProtocol = protocolHTTP
+		if targetProtocol != "" {
```

Comment:
Similarly, I don't think we need the _if_ statement since `targetProtocol` is guaranteed to be populated by `getTargetProtocol()`.